### ISSUE TITLE
Can't reproduce paper results

### ISSUE THREAD

**Issue by Jcastanyo** [paulotto/canfnet#4]
Hi!

First of all, congratulations on this excellent work!

I am trying to reproduce the results of your paper but, for the moment, I cannot do it. I am trying to train CanfNet with your training data but the training loss gets stuck around 0.6 value, the IFL loss is around 12, and when I evaluate the trained model on the test set, I get an MAE of 3N. 

I also encountered some problems executing the evaluate.py script, I got an error because NORM_DIS variable is not defined.

I don't know if the train and evaluate scripts are updated to the final version, could you help me or give me some tips to train your network? I would appreciate it.

Thanks in advance, regards,
Julio.

**Comment by paulotto** [comment]
Hi Julio,

Thank you for being interested. 

One reason I can imagine is that the training is very sensitive to the normalization parameters. So make sure that you choose the correct values in the train.py as well as evaluate.py scripts for either the Gelsight or DIGIT sensor. 

Furthermore, you could try to play a bit with the loss function parameters. 

Please note that the training and evaluation scripts were not initially part of the repository and were uploaded without checking their functionality. They may be updated in the future.  

I fixed the bug with the variable NORM_DIS in the evaluate.py script.

Best regards,
Paul
